swift/example_code/iam/README.md

```diff
@@ -0,0 +1,71 @@
+# Amazon IAM code examples for the SDK for Swift
+## Overview
+This folder contains code examples demonstrating how to use the AWS SDK for
+Swift to use the Amazon Identity and Access Management (`IAM`). This README
+discusses how to run these examples.
+
+Amazon Identity and Access Management (`Amazon IAM`) lets you specify who and
+what can access services and resources on your AWS account, centrally manage
+fine-grained permissions, and analyze access to refine permissions across AWS.
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Code examples
+
+### Single actions
+Code excerpts that show you how to call individual service functions.
+* [Attach an IAM policy to a
+  role](./AttachRolePolicy/Sources/ServiceHandler/ServiceHandler.swift).
+  (`AttachRolePolicy`)
+* [Create a new IAM user](./CreateUser/Sources/ServiceHandler/ServiceHandler.swift) (`CreateUser`)
+* [Create an IAM role](./CreateRole/Sources/ServiceHandler/ServiceHandler.swift) (`CreateRole`)
+* [Create an IAM role linked to a specific service](./CreateServiceLinkedRole/Sources/ServiceHandler/ServiceHandler.swift) (`CreateServiceLinkedRole`)
+* [Get information about an IAM role](./GetRole/Sources/ServiceHandler/ServiceHandler.swift) (`GetRole`)
+* [Get information about an IAM policy](./GetPolicy/Sources/ServiceHandler/ServiceHandler.swift) (`GetPolicy`)
+* [List the managed policies attached to a role](./ListAttachedRolePolicies/Sources/ServiceHandler/ServiceHandler.swift). (`ListAttachedRolePolicies`)
+* [List all groups on an AWS account](./ListGroups/Sources/ServiceHandler/ServiceHandler.swift) (`ListGroups`)
+* [List all AWS policies](./ListPolicies/Sources/ServiceHandler/ServiceHandler.swift) (`ListPolicies`)
+* [List the policies embedded in a role](./ListRolePolicies/Sources/ServiceHandler/ServiceHandler.swift). This does _not_ include managed policies attached to the role. (`ListRolePolicies`)
+* [List all users on an AWS account](./ListUsers/Sources/ServiceHandler/ServiceHandler.swift) (`ListUsers`)
```

Comment:
Consider organizing the links on lines 27-33 in alphabetical order: 
Get information about an IAM policy 
Get information about an IAM role
List all AWS policies
List all groups on an AWS account
List all users on an AWS account
List the managed policies attached to a role
List the policies embedded in a role